pkgs/applications/networking/browsers/elinks/default.nix

```
@@ -36,18 +36,24 @@ stdenv.mkDerivation {
     "--enable-cgi"
     "--enable-bittorrent"
     "--enable-nntp"
-    "--with-openssl=${openssl.dev}"
-    "--with-bzip2=${bzip2.dev}"
+    "--enable-256-colors"
+    "--with-libev"
   ] ++ stdenv.lib.optional enableGuile        "--with-guile"
     ++ stdenv.lib.optional enablePython       "--with-python"
     ++ stdenv.lib.optional enablePerl         "--with-perl"
     ++ stdenv.lib.optional enableSpidermonkey "--with-spidermonkey=${spidermonkey}"
     ;
+  
+  preConfigure = ''
+    patchShebangs ./autogen.sh
+    ./autogen.sh
+  '';
 
   meta = {
```

Currently ofborg fails to evaluate this, as `maintainers` isn't in scope.
```suggestion
  meta = with stdenv.lib; {
    description = "Full-featured text-mode web browser (package based on the fork felinks)";
    homepage = "https://github.com/rkd77/felinks";
    license = licenses.gpl2;
    platforms = platforms.linux;
    maintainers = with maintainers; [ iblech ];
  };
```

---

Oops, thank you, I added the maintainers field at the last moment and didn't notice that it wasn't in scope. I also tried `platforms.unix`, I cannot test on macOS, but Hydra can, let's see!